src/main/config/configuration.properties

```diff
@@ -14,6 +14,7 @@ cli.color = true
 plugindirs = plugins;demo
 plugin.ignores =
 plugin.order = org.dita.base org.oasis-open.dita.v1_3 org.oasis-open.dita.v1_2
+registry = https://raw.githubusercontent.com/jelovirt/dita-ot-index/master/
```

Comment:
The jelovirt/dita-ot-index repository needs to be moved to dita-ot ownership and this changed accordingly before merge.